<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-KFO1UG6" name="Banking App Test" tests="1" failures="1" timestamp="21 May 2021 08:49:15 GMT" time="35.143" errors="0">
  <testcase name="loginTest" time="6.538" classname="com.inetbanking.testcases.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.inetbanking.testcases.LoginTest.loginTest(LoginTest.java:31)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Banking App Test -->
